bldsys/cmake/sample_helper.cmake

```diff
@@ -181,4 +199,36 @@ endif()
     if(VKB_DO_CLANG_TIDY)
         set_target_properties(${PROJECT_NAME} PROPERTIES CXX_CLANG_TIDY "${VKB_DO_CLANG_TIDY}")
     endif()
+
+    if(DEFINED Vulkan_dxc_EXECUTABLE AND DEFINED SHADERS_HLSL)
+        compile_hlsl_shaders()
+    endif()
+endfunction()
+
+function(compile_hlsl_shaders)
+    foreach(SHADER_FILE_HLSL ${SHADERS_HLSL})
+        set(HLSL_SPV_FILE ${SHADER_FILE_HLSL}.spv)
+
+        if(${SHADER_FILE_HLSL} MATCHES "[^-]+.vert.hlsl")
+            set(DXC_PROFILE "vs_6_1")
+        elseif(${SHADER_FILE_HLSL} MATCHES "[^-]+.frag.hlsl")
+            set(DXC_PROFILE "ps_6_4")
+        elseif(${SHADER_FILE_HLSL} MATCHES "[^-]+.rgen..hlsl" OR ${SHADER_FILE_HLSL} MATCHES "[^-]+.rmiss.hlsl" OR ${SHADER_FILE_HLSL} MATCHES "[^-]+.rchit.hlsl")
+            set(DXC_PROFILE "lib_6_3")
+            set(DXC_TARGET "-fspv-target-env=vulkan1.1spirv1.4")
+        elseif(${SHADER_FILE_HLSL} MATCHES "[^-]+.comp.hlsl")
+            set(DXC_PROFILE "cs_6_1")
+        elseif(${SHADER_FILE_HLSL} MATCHES "[^-]+.geom.hlsl")
+            set(DXC_PROFILE "gs_6_1")
+        elseif(${SHADER_FILE_HLSL} MATCHES "[^-]+.tesc.hlsl")
+            set(DXC_PROFILE "hs_6_1")
+        elseif(${SHADER_FILE_HLSL} MATCHES "[^-]+.tese.hlsl")
+            set(DXC_PROFILE "ds_6_1")
+        elseif(${SHADER_FILE_HLSL} MATCHES "[^-]+.mesh.hlsl")
+            set(DXC_PROFILE "cs_6_1")
```

Comment:
Not 100% sure, but I think this should be `ms_6_6` instead. At least that's what I use for compiling mesh shaders in my samples.

Reply:
It was mistake from my side, fixed.